Plots/Contours/NCL_h_lat_6.py

```diff
@@ -0,0 +1,120 @@
+
+################################################################################
+# Plot:
+
+# Generate figure (set its size (width, height) in inches) and axes
+plt.figure(figsize=(8, 8))
+ax = plt.axes()
+
+# Set y-axis to have log-scale
+plt.yscale('log')
+
+# Specify which contours should be drawn
+levels = np.linspace(-55, 55, 23)
+
+# Plot contour lines
+lines = U.plot.contour(ax=ax,
+                       levels=levels,
+                       colors='black',
+                       linewidths=0.5,
+                       linestyles='solid',
+                       add_labels=False)
+
+# Draw contour labels and set their backgrounds to be white
+ax.clabel(lines, fmt='%d', levels=levels)
+[
+    txt.set_bbox(dict(facecolor='white', edgecolor='none', pad=1))
+    for txt in lines.labelTexts
+]
+
+# Plot filled contours
+colors = U.plot.contourf(ax=ax,
+                         levels=levels,
+                         cmap=gvcmap.BlWhRe,
+                         add_labels=False,
+                         add_colorbar=False)
+# Add colorbar
+plt.colorbar(colors,
+             ax=ax,
+             orientation='horizontal',
+             ticks=levels[1::2],
+             drawedges=True,
+             aspect=12,
+             shrink=0.7,
+             pad=0.1)
+
+# Use geocat.viz.util convenience function to set axes tick values
+# Set y-lim inorder for y-axis to have descending values
+gvutil.set_axes_limits_and_ticks(ax,
+                                 xticks=np.linspace(-60, 60, 5),
+                                 xticklabels=['60S', '30S', '0', '30N', '60N'],
+                                 ylim=ax.get_ylim()[::-1],
+                                 yticks=U["lev"])
+
+# Change formatter or else tick values will be in exponential form
+ax.yaxis.set_major_formatter(ScalarFormatter())
+
+# Use geocat.viz.util convenience function to add major tick lines with no
+# minor ticks
+gvutil.add_major_minor_ticks(ax=ax,
+                             x_minor_per_major=3,
+                             y_minor_per_major=1,
+                             labelsize=10)
+
+# Use geocat.viz.util convenience function to add titles and the pressure label
+gvutil.set_titles_and_labels(ax,
+                             lefttitle=U.long_name,
+                             lefttitlefontsize=14,
+                             righttitle=U.units,
+                             righttitlefontsize=14,
+                             ylabel=U.lev.long_name + " (" + U.lev.units + ")",
+                             labelfontsize=14)
+
+# Create second y-axis to show geo-potential height.
+# Currently we're using arbitrary values for height as we haven't figured out
+# how to make this work properly yet
+axRHS = ax.twinx()
+axRHS.set_ylim(0, 32)
+axRHS.set_yticks(np.arange(4, 32, 4))
+axRHS.tick_params(labelsize=10)
+axRHS.set_ylabel('Height (km)')
+axRHS.yaxis.label.set_size(14)
+
+# Force the plot to be square by setting the aspect ratio to 1
+ax.set_box_aspect(1)
```

Comment:
Locally, I get an Attribute Error of: `'AxesSubplot' object has no attribute 'set_box_aspect'`. Is this consistent on your side too @anissa111? I still have an output projection even with the Attribute Error.

Reply:
That's really odd because when I'm running it locally, I don't get a single warning or error.

Reply:
I am also not getting that error locally in my IDE or when I build the html, that is strange.

Reply:
I looked the error up and it looks like it may be due to a version of matplotlib I am using? Not sure completely as we should all be on the same version, but if it isn't happening for Anissa, then I think it's fine to ignore!

Reply:
I'm also using Spyder, so it could maybe be an IDE thing too (if you are both not using Spyder)

Reply:
I just recently switched over to PyCharm, so maybe. Did you remake the geocat-examples conda environment after we switched back to the previous version?

Reply:
For reference I recently rebuilt my conda environment, and I'm using Visual Studio Code as my IDE.